roofit/roofitcore/inc/RooFit/Detail/EvaluateFuncs.h

```diff
@@ -30,6 +30,15 @@ inline double gaussianEvaluate(double x, double mean, double sigma)
    return std::exp(-0.5 * arg * arg / (sig * sig));
 }
 
+inline double rooPolyVarEvaluate(double *coeffs, int size, int lowestOrder, double x)
```

Comment:
```suggestion
inline double polyVarEvaluate(double const *coeffs, int size, int lowestOrder, double x)
```

Please use `const`. Also, the other functions in this file don't have the `roo` prefix, so maybe also drop it here?